src/App.js

```diff
@@ -1,45 +1,17 @@
-import {useState, useEffect} from 'react';
+import {Route, Routes} from 'react-router-dom';
 import styled from 'styled-components';
-import Cards from './components/Cards/Cards';
-import CategoryList from './components/CategoryList/CategoryList';
-import db from './db';
-import {loadFromLocalStorage, writeToLocalStorage} from './util/LocalStorage';
+import HomePage from './pages/HomePage';
+import AnimationPage from './pages/AnimationPage';
 
 export default function App() {
-  const [filter, setFilter] = useState('Alle');
-
-  const [exercises, setExercise] = useState(() => {
-    const exercisesFromLocal = loadFromLocalStorage('exercises');
-    return exercisesFromLocal ?? db;
-  });
-
-  function handleFilter(stringToFilter) {
-    setFilter(stringToFilter);
-  }
-
-  function handleBookmark(id) {
-    setExercise(
-      exercises.map(exercise => {
-        if (exercise.id === id) {
-          return {...exercise, bookmarked: !exercise.bookmarked};
-        }
-        return exercise;
-      })
-    );
-  }
-
-  useEffect(() => {
-    writeToLocalStorage('exercises', exercises);
-  }, [exercises]);
+  const minSecs = {minutes: 5, seconds: 0};
```

Comment:
I am not sure about the naming here: I see that there are minutes and seconds in this constant, but it's not clear what's the purpose of it. Please consider another, more speaking name.